main.go

```diff
@@ -46,11 +52,14 @@ func BuildHTTPMux(arbiterURL, liveContentStoreURL, draftContentStoreURL string)
 }
 
 func main() {
-	httpMux := BuildHTTPMux(arbiterHost, liveContentStoreHost, draftContentStoreHost)
+	httpMux := BuildHTTPMux(arbiterURL, liveContentStoreURL, draftContentStoreURL, errbitNotifier)
 
 	requestLogger, err := request_logger.New(requestLogDest)
 	if err != nil {
 		log.Fatal(err)
+		if errbitNotifier != nil {
+			errbitNotifier.Notify(err, &http.Request{})
+		}
```

Comment:
[`log.Fatal` calls `os.Exit`](http://golang.org/pkg/log/#Fatal), so anything after it won't have any effect.  The errbit call will need to be moved before the log call.